internal/interopgrpc/test_server.go

```diff
@@ -29,13 +29,15 @@ import (
 	"time"
 
 	testpb "github.com/bufbuild/connect-crosstest/internal/gen/proto/go/grpc/testing"
-	"github.com/bufbuild/connect-crosstest/internal/interopconnect"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
 )
 
+// nonASCIIErrMsg is a non-ASCII error message.
+const nonASCIIErrMsg = "soirée 🎉" // readable non-ASCII
```

Comment:
cannot find of a good place to put this and share by both interopgrpc and interopconnect, considering that we wont actually have a lot of them, I think it might be better to just duplicate them